src/Parser/DemonHunter/Havoc/Modules/Items/DelusionsOfGrandeur.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SpellLink from 'common/SpellLink';
+import ItemLink from 'common/ItemLink';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import Wrapper from 'common/Wrapper';
+import FuryTracker from '../ResourceTracker/FuryTracker';
+
+/*
+* Equip: The remaining cooldown on Metamorphosis is reduced by 1 sec for every 30 Fury you spend.
+*/
+
+class DelusionsOfGrandeur extends Analyzer {
+	static dependencies = {
+		combatants: Combatants,
+		SpellUsable: SpellUsable,
+		furyTracker: FuryTracker,
+	}
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasShoulder(ITEMS.DELUSIONS_OF_GRANDEUR.id);
+	}
+
+	get percentCDRWasted(){
+		return this.furyTracker.cooldownReductionWasted / this.furyTracker.cooldownReduction;
+	}
+
+	get suggestionThresholds() {
+    return {
+      actual: this.percentCDRWasted,
+      isGreaterThan: {
```

Comment:
I don't really like 'CDR wasted' suggestions / stats, because in my opinion they're just a more confusing version of cast efficiency. Doesn't this suggestion do basically the exact same thing as a cast efficiency suggestion?